src/porklock/core.clj

```diff
@@ -71,13 +70,13 @@
 
       ["-e"
        "--exclude"
-       "List of files to be excluded from uploads."
+       "The path to a file containing a list of paths to be excluded from uploads."
        :default ""]
 
       ["-x"
        "--exclude-delimiter"
        "Delimiter for the list of files to be excluded from uploads"
-       :default ","]
+       :default "\n"]
```

Comment:
We might need to ask @tedgin but as I recall at one point we had filenames with newlines in them. Of course we probably also have files with commas in them. Would it maybe make sense to do the `xargs`/`parallel` thing here and delimit with a null byte?

Reply:
That might work. I'd be inclined to leave the newline here, especially for testing, but I wonder if `road-runner`, or whatever else is creating the file and calling `porklock`, could pass the null-char on that command-line for this `--exclude-delimiter` param.

Reply:
That could be good. Though I don't know for sure if it's even possible to pass a null byte on the command line, so we'd need to check that. In relation to the other comment, though, it'll be less useful for debugging if it's null characters too.

One more work-intensive option would be to let it just be newlines, but expect each value to be (potentially) quoted, to handle the "newlines in a path" issue -- i.e., basically, make it like a path-list/CSV file.

Reply:
> In relation to the other comment, though, it'll be less useful for debugging if it's null characters too.

Oh yeah, good point 👍 

Also, newlines in filenames will cause issues in our path lists as well.

Since the DE doesn't allow users to create or rename files with newlines in them, maybe we don't need to be concerned at this point, and those users with non-standard filenames can use these features at their own risk?

Reply:
Yeah, I think that'll probably be fine. This is a much safer character than the comma we were using before, certainly!